docs/src/guides/spring/redis.md

```diff
@@ -42,3 +42,37 @@ Please check the [Spring Common Application properties](https://docs.spring.io/s
 ## 4. Connect to Redis
 
 Commit that code and push. The Redis instance is ready to be connected from within the Spring application.
+
+## Use Spring Data for Redis
+
+You can use [Spring Data Redis](https://spring.io/projects/spring-data-mongodb) to use Redis with your app.
+First, determine the MongoDB client using the [Java configuration reader library](https://github.com/platformsh/config-reader-java).
```

Comment:
Can Spring only be used with the MongoDB client? Since we list MariaDB and Mysql I doubt it.
We also list MongoDB there:
https://pr-2212-imxy7bi-652soceglkw4u.eu-3.platformsh.site/guides/spring/rabbitmq.html

Reply:
I'd say details like that are slightly out of scope.

Reply:
Haha sorry, I tried to understand the details of the guides along the way...